core/client/src/client.rs

```diff
@@ -146,13 +151,17 @@ pub trait BlockchainEvents<Block: BlockT> {
 	/// Get storage changes event stream.
 	///
 	/// Passing `None` as `filter_keys` subscribes to all storage changes.
-	fn storage_changes_notification_stream(&self, filter_keys: Option<&[StorageKey]>) -> error::Result<StorageEventStream<Block::Hash>>;
+	fn storage_changes_notification_stream(&self,
```

Comment:
Is it a style choice to leave the `self` parameter on the same line? I usually just move each argument to its own line when breaking long lines.

Reply:
for the cases where the first argument is obviously special (e.g. `target: "somewhere"` or `self`) and doesn't conform to the same grammar as the other items, i think it's ok to have it on the same line.